### DISCUSSION TITLE
Plesk with SSL Certificate cannot handle .env port

### DISCUSSION THREAD

**Discussion by 3logy** [directus/directus#3400] [Archive]
Hello,

 After starting with _npx directus start_, the webserver which was already run with Directus 8.0 cannot be found. To make it accessible i have to remove the ssl protection and add the port (i.e 7600) to see directus. 
But i need the default port since Plesk set it up and ssl per default as well. 

How can i change the .env file to make it happend?

**Comment by rijkvanzanten** [comment]
>  the webserver which was already run with Directus 8.0 cannot be found

This sounds like an XY problem to me. If you start a new web project on port 80, the previous project on port 80 is gone. You can't have 2 projects on the same port. That's not necessarily a Directus specific problem..
